script.js

```diff
@@ -1,7 +1,350 @@
+/* eslint no-undef: "error" */
+
 const myFunction = () => {
   document.getElementById('drop-menu').classList.add('show');
-}
+  document.body.classList.add('overflow');
+};
 
 const myFunction1 = () => {
   document.getElementById('drop-menu').classList.remove('show');
-}
\ No newline at end of file
+  document.body.classList.remove('overflow');
+};
+
+const myFunction2 = () => {
+  document.getElementById('popup_desktop').classList.add('view');
+  window.scrollTo(0, 0);
+  document.body.classList.add('overflow');
+  document.getElementById('sect').classList.add('blur');
+};
+
+const myFunction3 = () => {
+  document.getElementById('popup_desktop').classList.remove('view');
+  window.scrollTo(0, 1000);
+  document.body.classList.remove('overflow');
+  document.getElementById('sect').classList.remove('blur');
+};
+
+const myFunction4 = () => {
+  document.getElementById('popup_mobile').classList.add('see');
+  window.scrollTo(0, 0);
+  document.body.classList.add('overflow');
+  document.getElementById('sect').classList.add('blur');
+};
+
+const myFunction5 = () => {
+  document.getElementById('popup_mobile').classList.remove('see');
+  window.scrollTo(0, 1000);
+  document.body.classList.remove('overflow');
+  document.getElementById('sect').classList.remove('blur');
+};
+
+const projects_container = [
+  {
+    title: 'Multi-Post Stories',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['css', 'html', 'bootstrap', 'Ruby'],
+    'image desktop': 'images/Img.png',
+    'image mobile': 'images/image1.png',
+    button: 'See Project',
+    'link to live version': '#1',
+    'link to source': '#2',
+    class: "hide"
+  },
+
+  {
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#3',
+    'link to source': '#4',
+    class: "projects_art"
+  },
```

Comment:
- [ ] As previously requested, each object should have at least a name, description, image, tech, and link keys. All of your objects except the first are missing some keys.

Reply:
Most of the projects have background images that have been applied to the CSS.

Reply:
Remember, this portfolio will have different projects that you'll create and they'll definitely have different images

Reply:
okay, I added the background images also.